DeveloperConsole/InitializePatch.cs

```diff
@@ -1,20 +1,17 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
-using Il2CppTLD.AddressableAssets;
 using UnityEngine;
-using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.AddressableAssets;
 
-namespace DeveloperConsole {
+namespace DeveloperConsole
+{
```

Comment:
I'm not a fan of Java style bracketing either, but you should always adhere to the existing style when contributing.

Reply:
my VS was auto formatting.. turned it off.